posthog/cdp/templates/google_ads/template_google_ads.py

```diff
@@ -58,27 +60,32 @@
             "secret": False,
             "required": True,
         },
-        {
-            "key": "developerToken",
-            "type": "string",
-            "label": "Developer token",
-            "description": "This should be a 22-character long alphanumeric string. Check out this page on how to obtain such a token: https://developers.google.com/google-ads/api/docs/get-started/dev-token",
-            "secret": False,
-            "required": True,
-        },
         {
             "key": "customerId",
-            "type": "string",
+            "type": "integration_field",
+            "integration_key": "oauth",
+            "integration_field": "google_ads_customer_id",
             "label": "Customer ID",
             "description": "ID of your Google Ads Account. This should be 10-digits and in XXX-XXX-XXXX format.",
             "secret": False,
             "required": True,
         },
         {
             "key": "conversionActionId",
+            "type": "integration_field",
+            "integration_key": "oauth",
+            "integration_field": "google_ads_conversion_action",
+            "required_field": "customerId",
+            "label": "Conversion action",
+            "description": "The Conversion action associated with this conversion.",
+            "secret": False,
+            "required": True,
+        },
+        {
+            "key": "developerToken",
             "type": "string",
-            "label": "Conversion action ID",
-            "description": "You will find this information in the event snippet for your conversion action, for example send_to: AW-CONVERSION_ID/AW-CONVERSION_LABEL. This should be in the AW-CONVERSION_ID format.",
+            "label": "Developer token",
```

Comment:
This one still throws me... Segment doesn't need this so why do we need to get it from the user? Is this just due to wanting to keep our token safe? If so then we should do a different solution tbh as this feels like a bad requirement to put on users.

Reply:
Exactly - removing the dev token requirement would be a huge unlock. Continuing on https://github.com/PostHog/posthog/pull/26133